src/Analysis/Ast/Test/AssignmentTests.cs

```diff
@@ -160,7 +160,16 @@ class test():
             const string code = @"
 x, y, z = func()
 ";
-            var analysis = await GetAnalysisAsync(code, PythonVersions.LatestAvailable3X);
+            var analysis = await GetAnalysisAsync(code);
+            analysis.Should().HaveVariable("x").And.HaveVariable("y").And.HaveVariable("z");
+        }
+
+        [TestMethod, Priority(0)]
+        public async Task TupleSingleValue() {
+            const string code = @"
+(x, (y, (z))) = 1,
```

Comment:
Did you choose this example for a specific reason?
I would use a working python example like "(x, (y, (z))) = (1, (1., ''))" and check for the types returned by analysis

Reply:
The re is another one added below. This one has error in the RHS and a single value so it is a separate example